.circleci/config.yml

```diff
@@ -1402,11 +1405,12 @@ workflows:
   AWS-Daily-Services-Comp-Restart-HTS-Performance-6N-6C:
     triggers:
       - schedule:
-          cron: "30 4 * * *"
+          cron: "15 20 * * *"
```

Comment:
I believe the cron job changes cam be reverted

Reply:
Yeah. I will keep this in mind. If there is no other places needs significant changes, I will post the cron job change to another issue to change/simplify config.yml that's in progress right now. There I need to redo the cron job time slots again to avoid conflicts of GCP resources.